src/libsystemd-network/sd-lldp.c

```diff
@@ -221,15 +215,14 @@ int lldp_handle_packet(tlv_packet *tlv, uint16_t length) {
                 case LLDP_TYPE_CHASSIS_ID:
 
                         if (len < 2) {
-                                log_lldp("Received malformed Chassis ID TLV len = %d. Dropping",
-                                         len);
+                                log_lldp("Received malformed Chassis ID TLV len = '%d'. Dropping.", len);
 
                                 malformed = true;
                                 goto out;
                         }
 
                         if (chassis_id) {
-                                log_lldp("Duplicate Chassis ID TLV found. Dropping ...");
+                                log_lldp("Duplicate Chassis ID TLV found. Dropping.");
```

Comment:
That's not the point. The packet that causes this problem was sent from some other host, and so we should at least log that host's MAC or something. What action is an administrator supposed to take when this message appears in the logs? What benefit does it have in its current form?